servers/quarkus-server/src/test/java/org/projectnessie/server/AbstractTestQuarkusRest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2022 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server;
+
+import java.net.URI;
+import org.junit.jupiter.api.BeforeEach;
+import org.projectnessie.jaxrs.AbstractTestRest;
+
+public abstract class AbstractTestQuarkusRest extends AbstractTestRest {
```

Comment:
actually it is not possible (see CI errors: https://github.com/projectnessie/nessie/actions/runs/3233984593/jobs/5296535131)
because sub-classes like `TestRestApiInMemory` use plain `@QuarkusTest`